addon/components/frost-toggle-button.js

```diff
@@ -0,0 +1,106 @@
+import Ember from 'ember'
+const {computed, Component, ViewUtils} = Ember
+import layout from '../templates/components/frost-toggle-button'
+import FrostEventsProxy from '../mixins/frost-events-proxy'
+
+export default Component.extend(FrostEventsProxy, {
+
+  // ==========================================================================
+  // Properties
+  // ==========================================================================
```

Comment:
I'm trying to do the single line format of these section headers now (saves space) - see text or password for an example